src/components/Cast/VerifyWallet.tsx

```diff
@@ -1,16 +1,44 @@
 import { BodyText, ErrorText } from 'components/ui/Text'
+import { useState } from 'preact/hooks'
+import Button from 'components/ui/Button'
 import LoadingBlock from 'components/ui/LoadingBlock'
 import SealSad from 'icons/SealSad'
 
 export default function ({
   text,
   status,
   error,
+  onButtonClick,
 }: {
   status: string
   text: string
   error?: unknown
+  onButtonClick?: () => void
 }) {
+  const [isGSNError, setIsGSNError] = useState(
+    error && typeof error === 'string' && error.includes('Failed to relay call')
```

Comment:
I think we should have a file with `string` errors like this (`enum` or `object` of those), we already have it in package

Reply:
Added a task to Trello for that